Source/WTF/wtf/text/AtomStringImpl.cpp

```diff
@@ -281,12 +281,12 @@ struct BufferFromStaticDataTranslator {
         return buf.hash;
     }
 
-    static bool equal(PackedPtr<StringImpl> const& str, const Buffer& buf)
+    static bool equal(typename AtomStringTable::StringEntry const& str, const Buffer& buf)
```

Comment:
I don’t understand why `typename` is needed here but not above.

Reply:
It seems that this is not necessary. Dropped. (I originally though it is necessary because this class is templatized).